server/src/main/java/umm3601/Controller.java

```diff
@@ -0,0 +1,15 @@
+package umm3601;
+
+import io.javalin.Javalin;
+
+public interface Controller {
+  /**
+   * Add routes for the server.
```

Comment:
@willster000 correctly observed that this comment makes it sound like you actually add the routes in the interface _which isn't true_. We should modify this to make it clearer that this is "done" and any additional work happens in classes that implement this interface.